sdk/communication/AzureCommunicationChat/Source/Signaling/CommunicationSignalingClient.swift

```diff
@@ -79,6 +79,7 @@ class CommunicationSignalingClient {
     func start() {
         selfHostedTrouterClient.withRegistrar(trouterUrlRegistrar)
         selfHostedTrouterClient.start()
+        selfHostedTrouterClient.setUserActivityState(UserActivityState.TrouterUserActivityStateActive)
```

Comment:
Does this have anything to do with the telemetry??

Reply:
No, this is unrelated & to enable connection status on trouter side. It doesn't change anything on SDK but we need it for an internal customer

Reply:
I can split this into separate PR but its not really related to any feature we are releasing, so just included it here.